react-ui/src/components/SignUpForm.js

```diff
@@ -68,22 +73,47 @@ render() {
                 </label>
                 <br></br>
                 <br></br>
-                <label for="role" className ="Title-Style2"> Select Role: </label>
-                <select name="role" className ="InputFields2" id="type">
+                <label for="role" className ="Title-Style2"> Select User Role: </label>
```

Comment:
User role is a dropdown form option - users are expected to select either Student or Worker when signing up.